src/prommis/uky/uky_flowsheet.py

```diff
@@ -2075,15 +2086,26 @@ def add_costing(m):
 
     # 4.3 is UKy Leaching - Tank Mixer
     L_tank_mixer_accounts = ["4.3"]
-    m.fs.L_tank_mixers = UnitModelBlock()
-    m.fs.L_tank_mixers.power = Var(initialize=4.74, units=units.hp)
-    m.fs.L_tank_mixers.power.fix()
-    m.fs.L_tank_mixers.costing = UnitModelCostingBlock(
+    m.fs.leach_mixer.power = Var(initialize=4.74, units=units.hp, bounds=(0, None))
+
+    @m.fs.leach_mixer.Constraint(L_tank_mixer_accounts)
+    def power_scaling_constraint(c, k):
+        return m.fs.leach_mixer.power == units.convert(
+            REE_costing_params["1"][k]["RP Value"]
```

Comment:
What does RP value stand for again?

Reply:
RP stands for Reference Parameter, it's the unit operation parameter value from the cost accounts / UKy report at commercial scale.